app/lib/bk/compat/parsers/bitrise/loader.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module BK
+  module Compat
+    # Bitrise workflow configuration loader
+    class Bitrise
+      STEP_KEY_REGEX = /([a-zA-Z-]*)@(\d*|0)(\.((\d*)|0)){0,3}/
```

Comment:
why the `\d*|0`? wouldn't it be easier to use `\d+`?

```suggestion
      STEP_KEY_REGEX = /([a-zA-Z-]*)@(v?\d+(\.\d+){0,3})?/
```

Reply:
Probably just a case of old cmd+c/v, as below though I think this was overthinking it anyway (unless we actually wanted to use the specific values of versions etc - as `String#split` works much better here anyway 👍